rts/Net/NetCommands.cpp

```diff
@@ -461,7 +461,7 @@ void CGame::ClientReadNet()
 
 					CPlayer* player = playerHandler.Player(playerID);
 					pckt >> player->name;
-
+vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvg
```

Comment:
nuthin but a G thang
```suggestion
vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
```